src/fmu/dataio/_model/root.py

```diff
@@ -268,7 +270,7 @@ def generate(
     ) -> dict[str, Any]:
         json_schema = super().generate(schema, mode=mode)
         json_schema["$schema"] = self.schema_dialect
-        json_schema["$id"] = "fmu_results.json"
+        json_schema["$id"] = FmuResultsSchema.url()
```

Comment:
Is it correct that the `id` is set to a url?

Reply:
For a top-level schema, yep: [ref.](https://www.learnjsonschema.com/2020-12/core/id/). Not the best naming...

Incidentally, this value is ovewritten here anyway: